mkFit/MkFinder.cc

```diff
@@ -350,7 +350,7 @@ void MkFinder::SelectHitIndices(const LayerOfHits &layer_of_hits,
       float dphi = calcdphi(dphi2, min_dphi);
 
       const float  r = std::sqrt(r2);
-      const float dr = nSigmaR*std::sqrt(std::abs(x*x*Err[iI].ConstAt(itrack, 0, 0) + y*y*Err[iI].ConstAt(itrack, 1, 1) + 2*x*y*Err[iI].ConstAt(itrack, 0, 1)) / r2);
+      const float dr = nSigmaR*std::sqrt(std::abs(x*x*Err[iI].ConstAt(itrack, 0, 0) + y*y*Err[iI].ConstAt(itrack, 1, 1) + 2*x*y*Err[iI].ConstAt(itrack, 0, 1)) / r2) + std::abs(0.5f*(L.m_layer_info->m_zmax-L.m_layer_info->m_zmin)*std::tan(Par[iI].ConstAt(itrack, 5, 0)));
```

Comment:
more specific on what I meant in https://github.com/trackreco/mkFit/pull/355#issuecomment-916538932

```suggestion
      const float dr = nSigmaR*std::sqrt(std::abs(x*x*Err[iI].ConstAt(itrack, 0, 0) + y*y*Err[iI].ConstAt(itrack, 1, 1) + 2*x*y*Err[iI].ConstAt(itrack, 0, 1)) / r2);
      const float dr_for_wsr = dr + std::abs(0.5f*(L.m_layer_info->m_zmax-L.m_layer_info->m_zmin)*std::tan(Par[iI].ConstAt(itrack, 5, 0)));
```
and then  below
```diff
-       XWsrResult[itrack] = L.is_within_r_sensitive_region(r, dr);
+      XWsrResult[itrack] = L.is_within_r_sensitive_region(r, dr_for_wsr);
```

Reply:
good point... I missed the fact that dr is not used just for WSR...